exec/vector/src/main/java/org/apache/drill/exec/record/metadata/MapBuilder.java

```diff
@@ -185,6 +192,26 @@ public MapBuilder resumeMap() {
     return (MapBuilder) parent;
   }
 
+  /**
+   * Depending on whether the parent is a schema builder or map builder
+   * we resume appropriately.
+   */
+  @Override
+  public void resume() {
+    if (Objects.isNull(parent))
```

Comment:
@mbeckerle I don't know why the checkstyle is telling you the wrong file, but here, you'll need braces as well as at line 203. 

ie:
```java
if (parent instanceof MapBuilder) {
      resumeMap();
}
```

Reply:
@mbeckerle Confirmed.  I successfully built your branch by adding the aforementioned braces.  I'll save you some additional trouble.  There's another check style violation in `DaffodilBatchReader`.  Drill doesn't like star imports for some reason.

Reply:
What is the maven command line to just make it run this checkstyle?

Reply:
I just built Drill using the following command:

```sh
mvn clean install -DskipTests
```

I think you can do just run the check style with:

```sh
mvn checkstyle:checkstyle
```